Clic&Farm-iOSTests/AddInterventionVCTableViewTests.swift

```diff
@@ -0,0 +1,191 @@
+    XCTAssertEqual(estimatedHeight, 150, "Estimated height should be 150")
+  }
+
+  func test_tableViewHeightForRowAndEstimatedHeightForRow_withSelectedInputsTableView() {
+    //Given
+    let indexPath = IndexPath(row: 0, section: 0)
+    let heightForRow = addInterventionVC.tableView(
+      addInterventionVC.selectedInputsTableView,
+      heightForRowAt: indexPath)
+    let estimatedHeight = addInterventionVC.tableView(
+      addInterventionVC.selectedInputsTableView,
+      estimatedHeightForRowAt: indexPath)
+
+    //Then
+    XCTAssertEqual(heightForRow, 110, "Height should be 110")
+    XCTAssertEqual(estimatedHeight, 110, "Estimated height should be 110")
+  }
+
+  func test_tableViewHeightForRowAndEstimatedHeightForRow_withSelectedMaterialsTableView() {
+    //Given
+    let indexPath = IndexPath(row: 0, section: 0)
+    let heightForRow = addInterventionVC.tableView(
+      addInterventionVC.selectedMaterialsTableView,
+      heightForRowAt: indexPath)
+    let estimatedHeight = addInterventionVC.tableView(
+      addInterventionVC.selectedMaterialsTableView,
+      estimatedHeightForRowAt: indexPath)
+
+    //Then
+    XCTAssertEqual(heightForRow, 80, "Height should be 80")
+    XCTAssertEqual(estimatedHeight, 80, "Estimated height should be 80")
+  }
+
+  func test_tableViewHeightForRowAndEstimatedHeightForRow_withSelectedEquipmentsTableView() {
+    //Given
+    let indexPath = IndexPath(row: 0, section: 0)
+    let heightForRow = addInterventionVC.tableView(
+      addInterventionVC.selectedEquipmentsTableView,
+      heightForRowAt: indexPath)
+    let estimatedHeight = addInterventionVC.tableView(
+      addInterventionVC.selectedEquipmentsTableView,
+      estimatedHeightForRowAt: indexPath)
+
+    //Then
+    XCTAssertEqual(heightForRow, 55, "Height should be 55")
+    XCTAssertEqual(estimatedHeight, 55, "Estimated height should be 55")
+  }
+
+  func test_tableViewHeightForRowAndEstimatedHeightForRow_withSelectedPersonsTableView() {
+    //Given
+    let indexPath = IndexPath(row: 0, section: 0)
+    let heightForRow = addInterventionVC.tableView(
+      addInterventionVC.selectedPersonsTableView,
+      heightForRowAt: indexPath)
+    let estimatedHeight = addInterventionVC.tableView(
+      addInterventionVC.selectedPersonsTableView,
+      estimatedHeightForRowAt: indexPath)
+
+    //Then
+    XCTAssertEqual(heightForRow, 65, "Height should be 65")
+    XCTAssertEqual(estimatedHeight, 65, "Estimated height should be 65")
+  }
+
+  func test_tableViewHeightForRowAndEstimatedHeightForRow_withRandomTableView() {
+    //Given
+    let tableView = UITableView()
+    let indexPath = IndexPath(row: 0, section: 0)
+    let heightForRow = addInterventionVC.tableView(tableView, heightForRowAt: indexPath)
+    let estimatedHeight = addInterventionVC.tableView(tableView, estimatedHeightForRowAt: indexPath)
+
+    //Then
+    XCTAssertEqual(heightForRow, 60, "Height should be 60")
```

Comment:
Are all those Height/EstimatedHeight tests testing the intervention's presence or are they testing the actual layout?
If it is the layout this isn't a great idea, it is better to test functionality, layouts/UIs might change.